DSL.Ruuter/POST/annotate_corpora.yml

```diff
@@ -0,0 +1,8 @@
+annotate_corpora:
+  call: http.post
+  args:
+    url: http://web:5001/annotate_corpora
```

Comment:
What endpoint is `http://web:5001/annotate_corpora`?

Reply:
This is the endpoint which is hosted on our Anonymiser Service. It pseudonymises a particular text using the trained model.